packages/datadog-core/src/utils/src/extract-package-and-module-path.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
```

Comment:
This doesn't really belong in core as it's only needed for auto-instrumentation. Maybe it could go in `datadog-instrumentations`?